yarn-project/scripts/src/benchmarks/aggregate.ts

```diff
@@ -66,6 +67,13 @@ function append(
   results[metric]![bucket].push(numeric);
 }
 
+/** Processes an entry with event name 'acir-proof-generated' and updates results */
+function processAcirProofGenerated(entry: ProofConstructed, results: BenchmarkCollectedResults) {
+  if (entry.acir_test == 'sha256') {
```

Comment:
```suggestion
  if (entry.acir_test === 'sha256') {
```
Because I'm annoying and I like to point something out on every PR

Reply:
tsk, javascript :disappointed: